pytest.ini

```diff
@@ -5,3 +5,16 @@ env =
     # pragma: allowlist nextline secret
     D:DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/fsd_account_store
     GITHUB_SHA=123123
+
+markers =
+    user_config: used to provide users to tests
+
+filterwarnings =
+    error
+
+    # ignore warnings from connexion
+    ignore::DeprecationWarning:connexion\.json_schema
+    ignore::DeprecationWarning:connexion\.validators
+
+    # ignore warnings from pytest about unraisable exceptions caused during test teardown
+    ignore::pytest.PytestUnraisableExceptionWarning
```

Comment:
This one is weird hmmm ... feels like one that should be in our control to fix 🤔

Reply:
This was quite tricky to solve. 

`flask_test_client` fixture is scoped `function`.
`clear_test_data` fixture is scoped for the `session`.

I am not even sure how it works when the connextion flask instance is scoped per function, but the refresh of the database is scoped per session. But it was causing that exception issue.

Reply:
Nice - well done for solving it 👍